build.gradle.kts

```diff
@@ -488,46 +514,25 @@ tasks {
     }
   }
 
-  patchPluginXml {
-    sinceBuild = properties("pluginSinceBuild")
-    untilBuild = properties("pluginUntilBuild")
-  }
-
   runIde {
     dependsOn(project.tasks.getByPath("buildCody"))
     jvmArgs("-Djdk.module.illegalAccess.silent=true")
 
     agentProperties.forEach { (key, value) -> systemProperty(key, value) }
 
-    val platformRuntimeVersion = project.findProperty("platformRuntimeVersion")
-    val platformRuntimeType = project.findProperty("platformRuntimeType")
-    if (platformRuntimeVersion != null || platformRuntimeType != null) {
-      val ideaInstallDir =
-          getIdeaInstallDir(
-              platformRuntimeVersion.or(project.property("platformVersion")).toString(),
-              platformRuntimeType.or(project.property("platformType")).toString())
-              ?: throw GradleException(
-                  "Could not find IntelliJ install for version: $platformRuntimeVersion")
-      ideDir.set(ideaInstallDir)
-    }
-    // TODO: we need to wait to switch to Platform Gradle Plugin 2.0.0 to be able to have separate
-    // runtime plugins
-    // https://github.com/JetBrains/intellij-platform-gradle-plugin/issues/1489
-    // val platformRuntimePlugins = project.findProperty("platformRuntimePlugins")
-  }
-
-  runPluginVerifier {
-    ideVersions.set(versionsToValidate)
-    failureLevel.set(EnumSet.complementOf(skippedFailureLevels))
-  }
+    //    val platformRuntimeVersion = project.findProperty("platformRuntimeVersion")
```

Comment:
@mkondratek I commented this out but we need to adjust it to new API.
Looks like `version` and `type` are now properties of the task.
https://plugins.jetbrains.com/docs/intellij/tools-intellij-platform-gradle-plugin-testing-extension.html#intellijPlatformTesting